Source/BSN.IpTables.Cli/custom/Connect-BsnIPTablesCli.ps1

```diff
@@ -0,0 +1,22 @@
+function Connect-BsnIPTablesCli {
```

Comment:
All cmdlet must have docs in form of powershell standard docs.

Reply:
Done.